sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -464,6 +464,16 @@ object SQLConf {
     .intConf
     .createWithDefault(20)
 
+  val DATA_SOURCE_TABLE_RELATION_CACHE_MAX_SIZE =
+    SQLConfigBuilder("spark.sql.sources.tableRelationCache.maxSize")
```

Comment:
ah, I hadn't noticed that, thanks for the reminder!
how about we name it `spark.sql.filesourceTableRelationCacheSize`?